tests/e2e/test_rosacli_cluster.go

```diff
@@ -328,9 +328,149 @@ var _ = Describe("Edit cluster",
 					"-y")
 				Expect(err).To(HaveOccurred())
 				textData = rosaClient.Parser.TextData.Input(output).Parse().Tip()
-				Expect(textData).
+        Expect(textData).
 					To(ContainSubstring(
-						"ERR: Schedule '\"5 5\"' is not a valid cron expression"))
+						"ERR: Schedule '\"5 5\"' is not a valid cron expression"))			
+      })
+
+		// Excluded until bug OCM-8408 is resolved
+		It("can verify delete protection on a rosa cluster - [id:73161]",
+			labels.High, labels.Runtime.Day2, labels.Exclude,
+			func() {
+				By("Enable delete protection on the cluster")
+				deleteProtection := "Enabled"
+				_, err := clusterService.EditCluster(clusterID,
+					"--enable-delete-protection=true",
+					"-y",
+				)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Check the enable result from cluster description")
+				output, err := clusterService.DescribeCluster(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+
+				clusterDetail, err := clusterService.ReflectClusterDescription(output)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(clusterDetail.EnableDeleteProtection).To(Equal(deleteProtection))
+
+				By("Enable delete protection with invalid values")
+				_, err = clusterService.EditCluster(clusterID,
+					"--enable-delete-protection=aaa",
+					"-y",
+				)
+				Expect(err).To(HaveOccurred())
+				textData := rosaClient.Parser.TextData.Input(output).Parse().Tip()
+				Expect(textData).Should(ContainSubstring(
+				`Error: invalid argument "aaa" for "--enable-delete-protection" flag: strconv.ParseBool: parsing "aaa": invalid syntax`))
+
```

Comment:
Negative checkpoints, let's have another case with Medium importance rather than put in together with positive one.